layers/+lang/coq/README.org

```diff
@@ -123,6 +123,12 @@ The mnemonic for =a= is "ask".
 | ~SPC m i t~     | Insert a tactic                                                 |
 | ~SPC m i T~     | Insert a tactical                                               |
 
+** Options
+
+| Key binding     | Description                 |
+|-----------------+-----------------------------|
+| ~SPC m o .~     | Toggle electric terminator. |
```

Comment:
Please change the keybinding to `SPC m T t`.
`SPC m T` is reserved for major mode specific toggles, that's also the place where all toggles for the layer should be kept.
`.` is also reserved, the conventions document says it needs to start a transient state i.e. `SPC w .` starting the window transient state.

See below the areas of CONVENTIONS.org I am refering to:

> *** Transient-state
Transient states are generally put behin the dot key `.`, for instance the
windows manipulation transient state in `SPC w .`.
When it is not possible to use the `.` (like in =helm= buffers) transient states
should be enabled with `M-SPC` and `s-M-SPC`. We need the latter bindings on
macOS since `M-SPC` is used by the OS for spotlight.
It is recommended to make sure that `q` allows to leave the transient-state.

> ** Toggles
Global toggles are under `SPC t`, `SPC T` and `SPC C-t`
Major mode toggles are only under `SPC m T`